openssl/src/dh.rs

```diff
@@ -1,3 +1,17 @@
+//! dh.rs
+//! 
+//! summary: bindings for dh library of OpenSSL
+//! author: steven fackler
+//! version: 
```

Comment:
I don't think the first couple of lines here are really relevant for someone reading this documentation.

Reply:
+1